src/main.tsx

```diff
@@ -1,22 +1,50 @@
 import "./css/style.scss";
 
 import { render } from "preact";
-import { banner, accordion } from "identity-style-guide";
+import { banner, accordion, navigation } from "identity-style-guide";
 import { Link } from "./router";
 import { Routes } from "./routes";
 
-[banner, accordion].forEach((component) => component.on());
+[banner, accordion, navigation].forEach((component) => component.on());
 
 render(
   <div>
-    <nav>
-      <div>
-        <Link href="/">Home</Link>
+    <header className="usa-header usa-header--extended">
```

Comment:
Worth moving into a component?

Reply:
🤷 it's only used once and doesn't contain any logic? Happy to move if you feel strongly, will leave in place for now